card/mod/02_basic_types/spec/set/all/all_csv_spec.rb

```diff
@@ -2,7 +2,7 @@
 
 describe Card::Set::All::AllCsv, "CSV mod" do
   context "csv_row view" do
-    it "should handle inclusions" do
+    it "should handle nests" do
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.